kfs-core/src/main/resources/edu/arizona/kfs/fp/spring-fp.xml

```diff
@@ -28,6 +28,10 @@
                 <value>edu/arizona/kfs/fp/ojb-fp.xml</value> 
             </list>
         </property>
+        <property name="triggerNames">
```

Comment:
There aren't actually any triggers defined in spring-fp, so this is an unneeded override.